class-udm-updater.php

```diff
@@ -590,11 +610,22 @@ public function admin_footer() {
 										console.log(resp);
 									}
 								} else if (resp.code == 'OK') {
-									alert('<?php echo esc_js(__('You have successfully connected for access to updates to this plugin.', 'udmupdater'));?>');
-									$('.udmupdater_box_<?php echo esc_js($this->slug);?>').parent().slideUp();
+									if (resp.hasOwnProperty('data') && resp.data.hasOwnProperty('plugin_info') && resp.data.plugin_info.hasOwnProperty('x-spm-duplicate-of')) {
+										alert($udmupdaterl10n.duplicate_site_id1);
+										$('div.udmupdater_box_<?php echo esc_js($this->slug);?> div.udmupdater_duplicate_site_warning').empty().text($udmupdaterl10n.duplicate_site_id2);
+										$('div.udmupdater_box_<?php echo esc_js($this->slug);?> div.udmupdater_duplicate_site_warning').slideDown();
+									} else {
+										alert('<?php echo esc_js(__('You have successfully connected for access to updates to this plugin.', 'udmupdater'));?>');
+										$('div.udmupdater_box_<?php echo esc_js($this->slug);?> div.udmupdater_duplicate_site_warning').empty();
+										$('.udmupdater_box_<?php echo esc_js($this->slug);?>').parent().slideUp();
+									}
 								} else if (resp.code == 'ERR') {
 									alert('<?php echo esc_js(__('Your login was accepted, but no available entitlement for this plugin was found.', 'udmupdater').' '.__('Has your licence expired, or have you used all your available licences elsewhere?', 'udmupdater'));?>');
 									console.log(resp);
+								} else {
+									if (resp.hasOwnProperty('data') && resp.data.hasOwnProperty('plugin_info') && resp.data.plugin_info.hasOwnProperty('duplicate_site_is_connected')) {
+										alert($udmupdaterl10n.duplicate_site_id3);
```

Comment:
@andtown I see that the message `duplicate_site_id3` says that a network timeout occurred, but I don't understand how we know that in this particular bit of the code??

Reply:
@DavidAnderson I've answered this question in this PR https://github.com/DavidAnderson684/simba-plugin-manager-updater/pull/12